src/theme/components/header/headerLowerNav.ts

```diff
@@ -29,6 +34,14 @@ const HeaderLowerNav = {
         ...headerFocus,
         ...linkFocusHoverStyles,
       },
+      _dark: {
+        color: "dark.ui.typography.heading",
+        _hover: linkFocusHoverDarkStyles,
```

Comment:
The `linkFocusHoverDarkStyles` object is the same as `linkFocusHoverStyles`, and because `color: "brand.primary"` was removed, you can replace `linkFocusHoverDarkStyles` with `linkFocusHoverStyles` here and on line 42 and remove the object.